.github/workflows/ci.yml

```diff
@@ -17,11 +17,14 @@ jobs:
           - { version: 4.9 }
     container:
       image: gcc:${{ matrix.build_config.version }}
+      options: -v /usr/local:/host_usr_local
     name: "gcc-${{ matrix.build_config.version }}"
     steps:
       - uses: actions/checkout@master
       - name: Setup
-        run: script/ci_setup_linux.sh
+        run: |
+          echo "/host_usr_local/bin" >> $GITHUB_PATH
```

Comment:
`GITHUB_PATH` is an env variable pointing to a file that you can use to share `PATH` changes across steps in the default runner.

This does not get automatically shared with containers and mounted. It's best to do it the way I commented earlier:
```yml
run: |
  PATH="$PATH:/host_usr_local/bin"
  script/ci_build.sh
```

Reply:
But then I have to add it to each step using cmake too. Is there a way to set it globally for each job?

Reply:
According to the [docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainerenv), `container` does support an `env`. Is it a bad idea to append the path there?

Reply:
Odd, even the old GCC / Clang builds use CMake 3.19.1. Maybe the env change is passed to the Image?

Reply:
Ah, good to know!